### PR TITLE
RHIROS-1134 Memory unit conversions

### PR DESCRIPTION
**Pull request by saltgen**
- [x] Convert memory unit from bytes to MiB in API responses
- [x] Update kruizepayload memory unit as bytes

### CODE REVIEW

File: internal/utils/utils.go [hunk 1]
```diff
@@ -156,3 +157,65 @@ func Start_prometheus_server() {
 		_ = http.ListenAndServe(fmt.Sprintf(":%s", cfg.PrometheusPort), nil)
 	}
 }
+
+func ConvertMemoryFromBytesToMiB(jsonData datatypes.JSON) map[string]interface{} {
```
Comment (patilsuraj767) [inline internal/utils/utils.go:161]: Please shift this function to `utils.go` which is under the `api` directory.

**Review by patilsuraj767 — CHANGES_REQUESTED** [review]

File: internal/api/handlers.go [hunk 2]
```diff
@@ -133,7 +135,7 @@ func GetRecommendationSet(c echo.Context) error {
 		recommendationSlice["workload"] = recommendationSet.Workload.WorkloadName
 		recommendationSlice["container"] = recommendationSet.ContainerName
 		recommendationSlice["last_reported"] = recommendationSet.Workload.Cluster.LastReportedAtStr
-		recommendationSlice["recommendations"] = recommendationSet.Recommendations
+		recommendationSlice["recommendations"] = utils.ConvertMemoryFromBytesToMiB(recommendationSet.Recommendations)
```
[the remark this answers is not in the document]
Comment (saltgen) [reply internal/api/handlers.go:138]: @patilsuraj767 Does `UpdateMemoryFromBytesToMiB` sound better?

[the remark this answers is not in the document]
Comment (saltgen) [reply internal/api/handlers.go:138]: I think the expectation is the complete recommendation object when the assignment is to `recommendationSlice["recommendations"] = `, but we can do better with the util function naming.

[the remark this answers is not in the document]
Comment (saltgen) [reply internal/api/handlers.go]: Done

File: internal/utils/utils.go (hunk 1 above)
Comment (saltgen) [reply internal/utils/utils.go]: Done

**Review by patilsuraj767 — CHANGES_REQUESTED** [review]
I tested the PR locally, `config` object looks good but we will also need to convert the `variation` object. 
Example - 
```
"short_term": {
                        "config": {
                            "limits": {
                                "cpu": {
                                    "amount": 0.06,
                                    "format": "cores"
                                },
                                "memory": {
                                    "amount": 493.99,
                                    "format": "MiB"
                                }
                            },
                            "requests": {
                                "cpu": {
                                    "amount": 0.06,
                                    "format": "cores"
                                },
                                "memory": {
                                    "amount": 493.99,
                                    "format": "MiB"
                                }
                            }
                        },
                        "duration_in_hours": 24,
                        "monitoring_end_time": "2023-06-03T00:45:00Z",
                        "monitoring_start_time": "2023-06-02T00:30:00Z",
                        "pods_count": 1,
                        "variation": {
                            "limits": {
                                "cpu": {
                                    "amount": -0.94,
                                    "format": "cores"
                                },
                                "memory": {
                                    "amount": -555755520,
                                    "format": "bytes"
                                }
                            },
                            "requests": {
                                "cpu": {
                                    "amount": -0.94,
                                    "format": "cores"
                                },
                                "memory": {
                                    "amount": -555755520,
                                    "format": "bytes"
                                }
                            }
                        }
                    }
```

**Review by patilsuraj767 — APPROVED** [review]
Thanks @saltgen PR looks good 👍

**Pull request merged**